test/full-setup/config.yml

```diff
@@ -46,13 +46,19 @@ domains:
         host: example.com
         health_check:
           scheme: http
+    jwt:
+      jwks_refresh_interval: 60
+      jwks_url: http://testJwksUrlYaml.com
 
   example_org:
     server:
       upstream:
         host: example.org
         health_check:
           scheme: http
+    jwt:
+      excluded_paths:
+        - /
```

Comment:
I believe that if you set these 2 lines under the block on line 49, it'll be inherited by all the domains, without the need of overriding the same value.